build.gradle

```diff
@@ -37,6 +37,8 @@ buildscript {
     classpath "org.owasp:dependency-check-gradle:$versions.owaspDepCheckPlugin"
     classpath "com.diffplug.spotless:spotless-plugin-gradle:$versions.spotlessPlugin"
     classpath "gradle.plugin.com.github.spotbugs:spotbugs-gradle-plugin:$versions.spotbugsPlugin"
+    // For the japicmp lib, required here for intellij
+    classpath "com.github.siom79.japicmp:japicmp:$versions.japicmp"
```

Comment:
This file doesn't need japicmp, right?

Reply:
Yep. I put it here because IntelliJ basically won't be able to resolve the dependency if it's only in compatibility.gradle and underlines everything although it's working in gradle. So I guess it's a gradle bug.